cedar-wasm/src/formatter.rs

```diff
@@ -0,0 +1,76 @@
+use cedar_policy_formatter::{policies_str_to_pretty, Config};
+use serde::{Deserialize, Serialize};
+
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Tsify, Debug, Serialize, Deserialize)]
+#[tsify(from_wasm_abi, into_wasm_abi)]
+pub struct FormattingResult {
+    success: bool,
+    #[serde(rename = "formattedPolicy", skip_serializing_if = "Option::is_none")]
+    formatted_policy: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    error: Option<String>,
+}
+
+#[wasm_bindgen(js_name = "formatPolicies")]
+pub fn wasm_format_policies(
+    policies_str: &str,
+    line_width: i32,
```

Comment:
Can you explain a bit more why we have 3 types for these (`i32`, `usize` and `isize`). I'm guessing `i32` is what wasm uses. Can we just use that everywhere here?

Reply:
The `cedar_policy_formatter::Config` type requires `usize` and `isize`. It seems like I can get away with making the inputs those types and not converting from `i32`, even though wasm only has `i32` and `i64` types internally. `wasm-pack` seems to build the function such that it works with the normal numbers in javascript. Thoughts on this @Swolebrain ?

Reply:
wouldn't that be a breaking change to the formatter though?

Reply:
Got it. I don't see why `indent_width` would be negative, but you're right that it would be a breaking change. Resolving.

Reply:
The reason is that the library we used to implement the formatter makes it that way. So I just propagated the behavior.